src/main/kotlin/uk/gov/dluhc/printapi/rds/mapper/RdsElectoralRegistrationOfficeMapper.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.dluhc.printapi.rds.mapper
+
+import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import uk.gov.dluhc.printapi.dto.EroManagementApiEroDto
+import uk.gov.dluhc.printapi.rds.entity.ElectoralRegistrationOffice
+
+@Mapper
+interface RdsElectoralRegistrationOfficeMapper {
```

Comment:
Prefix is so Spring Application context can start up without having duplicate bean names.